pyasic/data/pools.py

```diff
@@ -19,17 +50,17 @@ class PoolMetrics:
     pool_stale_percent: Percentage of stale shares by the pool.
     """
 
+    pool_rejected_percent: float = field(init=False)
+    pool_stale_percent: float = field(init=False)
+    url: PoolUrl
     accepted: int = None
     rejected: int = None
     get_failures: int = None
     remote_failures: int = None
     active: bool = None
     alive: bool = None
-    url: str = None
     index: int = None
     user: str = None
-    pool_rejected_percent: float = field(init=False)
```

Comment:
These dont need to be removed?

Reply:
I dont understand what you mean. can you give more clarity please.